sql_user_check_sap_hana/README.md

```diff
@@ -0,0 +1,29 @@
+Please run each of the .sql files on that database with the account credentials that you need to connect to LightBeam.
+
+Prerequisites:
+- Install [hdbsql](https://tools.hana.ondemand.com/#hanatools) on the machine.[Instruction](https://developers.sap.com/tutorials/hana-clients-install..html)
+
+Fetch Schema List:
+```shell
+hdbsql -n <DATABASE_HOST_IP>:<PORT> -u <DATABASE_USER> -p <PASSWORD> -d <DATABASE_NAME> "SELECT SCHEMA_NAME FROM SCHEMAS" > schema_list
```

Comment:
shouldn't this be database_stats.sh instead of hdbsql ??

Reply:
this is command for testing if hdbsql is installed and working